lib/src/values/simple_config_object.cc

```diff
@@ -264,6 +264,24 @@ namespace hocon {
         return keys;
     }
 
+    shared_ptr<simple_config_object> simple_config_object::empty() {
+        return empty_instance();
+    }
+
+    shared_ptr<simple_config_object> simple_config_object::empty(shared_origin origin) {
+        if (origin == nullptr) {
+            return empty();
+        } else {
+            // TODO: do this correctly
```

Comment:
@MikaelSmith I forgot to put WIP on this but I'm gonna be going through some of these today, will decided what to do with this.